sdk/cpp/core/src/codec_service.cpp

```diff
@@ -25,10 +25,10 @@
 
 #include "codec_provider.hpp"
 #include "codec_service.hpp"
-#include "logger.hpp"
 #include "entity_data_node_walker.hpp"
 #include "path_api.hpp"
 #include "types.hpp"
+#include "logger.hpp"
```

Comment:
Because `logger.hpp` uses `Python.h`, and `Python.h` does [pre-processing](https://docs.python.org/3.6/extending/extending.html#a-simple-example) conflicts with std headers for some platform(see [here](https://trac.macports.org/ticket/44288#no2) and [here](https://github.com/mwydmuch/ViZDoom/issues/56)). Python documentation recommends to put `Python.h` before any std libraries. I tried to put `Python.h`(included in `logger.hpp`) both [before](https://travis-ci.org/psykokwak4/ydk-gen/jobs/210817722) and [after](https://travis-ci.org/psykokwak4/ydk-gen/jobs/210855369)(since we are not using functions in python header extensively) the inclusion of every std libraries, and it seems like put `Python.h` after the inclusion of std libraries is the easier way. So I rearranged the ordering of headers of files include `logger.hpp`.